### PR TITLE
Add security notice about openstack service separation (SCRD-8969)

### PR DESCRIPTION
**Pull request by jeremy-moffitt**
pulled in PR from Bryans fork at https://github.com/bryanstephenson/doc-cloud/pull/1

### CODE REVIEW

File: xml/security-security_features.xml [hunk 1]
```diff
@@ -6,15 +6,43 @@
  <!ENTITY % entities SYSTEM "entity-decl.ent"> %entities;
 ]>
 <chapter xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" xml:id="newsecurity">
- <title>&productnamereg;: Security Features Overview</title>
+ <title>&productnamereg;: Security Planning and Features</title>
+ <section xml:id="idg-all-security-security_planning-xml-1">
+  <title>Security Planning</title>
+  <para>
+   &productnamereg; is a complex system which has many options which
```
Comment (csymons-suse) [inline xml/security-security_features.xml]: *that* has many options (will read better)

File: xml/security-security_features.xml [hunk 2]
```diff
@@ -6,15 +6,43 @@
...
+   affect the security of the system and the workloads running within it. Careful planning
+   is required to ensure the correct options and features are chosen for your particular
+   environment. You should read
+   <link xlink:href="https://docs.openstack.org/security-guide/introduction/security-boundaries-and-threats.html">Security Boundaries and Threats</link>
```
Comment (csymons-suse) [inline xml/security-security_features.xml]: s/You should read/Read/

File: xml/security-security_features.xml [hunk 3]
```diff
@@ -6,15 +6,43 @@
...
+   to understand how OpenStack defines security domains. Ensure that the network layout
+   you choose for your deployment provides adequate network separation for your specific
+   requirements. The other parts of the
+   <link xlink:href="https://docs.openstack.org/security-guide/">OpenStack Security Guide</link>
+   are also informative.
```
Comment (csymons-suse) [inline xml/security-security_features.xml]: This sentence strikes me funny. Presumably everything in the guide is informative. 
Suggest: We suggest careful study of the full (link)OpenStack Security Guide(/link).

File: xml/security-security_features.xml [hunk 4]
```diff
@@ -6,15 +6,43 @@
...
+   Take advantage of the security features described below in this section. 
+  </para>
+  <para>
+   OpenStack service processes need to run some commands as root in order to provide
+   functionality. The service processes run as non-root users but can escalate privilege
+   to root to perform certain operations. If a service process is compromised to the
+   extent that an attacker can execute arbitrary commands as the service user then it is
```
Comment (csymons-suse) [inline xml/security-security_features.xml]: as the service user, then

File: xml/security-security_features.xml [hunk 5]
```diff
@@ -6,15 +6,43 @@
...
+   likely that the attacker can elevate privilege to root and take over the system on
+   which the service is running. When planning the security controls for an OpenStack
+   deployment you should consider the services to be running as root when deciding which
```
Comment (csymons-suse) [inline xml/security-security_features.xml]: deployment,

File: xml/security-security_features.xml [hunk 6]
```diff
@@ -6,15 +6,43 @@
...
+   security controls are appropriate for your environment. Timely installation of
+   security patches is essential. Other controls such as Intrusion Prevention Systems
+   and Web Application Firewalls should be considered.
```
Comment (csymons-suse) [inline xml/security-security_features.xml]: ...Firewalls are recommended.

File: xml/security-security_features.xml [hunk 7]
```diff
@@ -6,15 +6,43 @@
...
  <section xml:id="idg-all-security-security_features-xml-6">
-  <title>Security features in &product;</title>
+  <title>Security Features in &product;</title>
   <para>
    Enterprises need protection against security breaches, insider threats, and
    operational issues that increase the risk to sensitive data. &product;
    provides capabilities that help you
    protect your data at rest and in transit, enable centralized key management,
-   and comply with the Payment Card Industry Data Security Standard (PCI-DSS).
+   and meet compliace requirements.
```
Comment (csymons-suse) [inline xml/security-security_features.xml]: s/compliace/compliance/

**Comment by jeremy-moffitt** [comment]
tried to push fixes for Carls feedback to Bryans PR, but I dont have write permissions on the fork

File: xml/security-security_features.xml (hunk 3 above)
Comment (dirkmueller) [reply xml/security-security_features.xml]: addressed in https://github.com/SUSE-Cloud/doc-cloud/pull/950

File: xml/security-security_features.xml (hunk 4 above)
Comment (dirkmueller) [reply xml/security-security_features.xml]: addressed in https://github.com/SUSE-Cloud/doc-cloud/pull/950

File: xml/security-security_features.xml (hunk 5 above)
Comment (dirkmueller) [reply xml/security-security_features.xml]: addressed in https://github.com/SUSE-Cloud/doc-cloud/pull/950

File: xml/security-security_features.xml (hunk 7 above)
Comment (dirkmueller) [reply xml/security-security_features.xml]: fixed in https://github.com/SUSE-Cloud/doc-cloud/pull/950

File: xml/security-security_features.xml (hunk 6 above)
Comment (dirkmueller) [reply xml/security-security_features.xml]: changed in https://github.com/SUSE-Cloud/doc-cloud/pull/950

File: xml/security-security_features.xml (hunk 1 above)
Comment (dirkmueller) [reply xml/security-security_features.xml]: done in https://github.com/SUSE-Cloud/doc-cloud/pull/950

**Comment by dirkmueller** [comment]
*sigh*, great that maintainers permissions are not granted just for the one thing that is urgent. Forked into my space and created as new PR in https://github.com/SUSE-Cloud/doc-cloud/pull/950

**Pull request closed without merging**